packages/lesswrong/lib/collections/userTagRels/newSchema.ts

```diff
@@ -11,6 +11,7 @@ const schema = {
     database: {
       type: "VARCHAR(27)",
       foreignKey: "Tags",
+      nullable: false,
```

Comment:
We don't use these and don't have any in the DB so this one could be wrong; let me know if this change needs to be undone.